app/assets/javascripts/show-free-text-question.js

```diff
@@ -0,0 +1,31 @@
+window.GOVUKAdmin = window.GOVUKAdmin || {};
+window.GOVUKAdmin.Modules = window.GOVUKAdmin.Modules || {};
+
+(function (Modules) {
+  "use strict";
+
+  Modules.ShowFreeTextQuestion = function () {
+    this.start = function($element) {
+      function addEventListener() {
+        var $yesRadio = $element.find('.yes-radio');
+        var $noRadio = $element.find('.no-radio');
+        var $textBox = $element.find('.free-text-question');
+
+        if ($yesRadio.length) {
+          $yesRadio.click(function(e) {
+            $textBox.removeClass('if-js-hide');
+          });
+        }
+
+        if ($noRadio.length) {
+          $noRadio.on('click', function(e) {
+            $textBox.addClass('if-js-hide');
+          });
+        }
+      }
+
+      addEventListener();
```

Comment:
I personally prefer keeping method invocations at the top of the method, so it's clearer what's happening when you read the file.